nikola/image_processing.py

```diff
@@ -117,13 +117,13 @@ def resize_image(self, src, dst, max_size, bigger_panoramas=True, preserve_exif_
             # Rotate according to EXIF
             value = exif['0th'].get(piexif.ImageIFD.Orientation, 1)
             if value in (3, 4):
-                im = im.rotate(180)
+                im = im.transpose(Image.ROTATE_180)
             elif value in (5, 6):
-                im = im.rotate(270)
+                im = im.transpose(Image.ROTATE_270)
             elif value in (7, 8):
-                im = im.rotate(90)
+                im = im.transpose(Image.ROTATE_90)
             if value in (2, 4, 5, 7):
-                im = ImageOps.mirror(im)
+                im = im.transpose(Image.FLIP_LEFT_RIGHT)
```

Comment:
This means you can delete the ImageOps import, I think (Travis tests will fail if you don't)